### PR TITLE
Adding Odd Networks

### PR DESCRIPTION
**Pull request by blainsmith**
Thanks for starting the list!

### CODE REVIEW

**Comment by n8225** [comment]
Looks good, thanks!

**Comment by nodiscc** [comment]
@blainsmith Is it released yet and where is it downloadable?

**Comment by blainsmith** [comment]
Yes it is. It is installable from npmjs.com. Do you need more info placed on my PR to indicate that? Demo/sample links as well?

**Comment by n8225** [comment]
I'm not sure how @nodiscc missed that.  If you have Demo site that would be great, but your current commit is sufficient.

File: README.md [hunk 1]
```diff
@@ -623,6 +623,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [Flumotion](http://flumotion.net/) - A streaming media server with intuitive graphical administration tools. ([Source Code](https://github.com/Flumotion/flumotion)) `LGPLv2` `Python`
   * [Hellowlol HTPC Manager fork](https://github.com/Hellowlol/HTPC-Manager) - A fully responsive interface to manage all your favorite media on your HTPC. ([Source Code](https://github.com/styxit/HTPC-Manager)) `MIT` `Python`
   * [Icecast 2](http://www.icecast.org/) - streaming audio/video server which can be used to create an Internet radio station or a privately running jukebox and many things in between. ([Source Code](https://git.xiph.org/?p=icecast-server.git;a=summary)) `GPLv2` `C`
+  * [Odd Networks](http://oddnetworks.com) - Open source video management system (VMS) and API for collections and videos with supporting SDKs for Roku, Apple iOS/tvOS, Google Androind, and Amazon FireTV. ([Source Code](https://github.com/oddnetworks/oddworks)) `MIT` `Nodejs`
```
Comment (Chouchen) [inline README.md]: *Android* instead of **Androind**

**Comment by blainsmith** [comment]
Fixed the typo. There is no live running demo, but there is a sample server that anyone can get running if you want that added.

**Pull request merged**